docs/release-notes_exporter_v2.0.0-rc.1.md

```diff
@@ -0,0 +1,255 @@
+>     - In *archive* mode, the same path is backed by the new v2 pipeline and returns a retro‑compatible `data` entry plus an additional `errors` array; processing is **best‑effort** and returns all valid results when possible. If a request yields **no valid results** and **at least one meaningful error**, the API returns an HTTP error.
+
+---
+
+## Overview
+
+Exporter v2 introduces **full duty tracing** (consensus rounds, pre/post partial signatures), **committee‑level views**, richer filtering, OpenAPI definitions, and a clearer configuration model with **two operating modes**:
+
+- **archive** — full traces persisted to disk, new `/traces/*` endpoints, and `/decideds` served by the v2 pipeline (adds `errors` array).
+- **standard** — v1‑compatible participants only; the legacy `/decideds` payload is preserved and old data is pruned automatically.
+
+v2 uses **best‑effort responses**: it returns partial results alongside non‑fatal issues in an `errors` array. If a request yields **no valid data** and **at least one meaningful error**, it returns an HTTP error instead of an empty success.
+
+---
+
+## Quick Start
+
+### 1) Choose a mode and enable the exporter
+
+```yaml
+# config.yaml (excerpt)
+exporter:
+  Enabled: true              # enable exporter pipeline
+  Mode: archive              # "archive" | "standard"
+  RetainSlots: 50400         # used in standard mode only (~7 days at 12s/slot)
+```
+
+**Archive mode requirements & effects**
+- Persists detailed traces on disk (Pebble). Higher CPU/disk and P2P traffic (subscribes to all subnets).
+- Enables new endpoints under `/v1/exporter/traces/*` and serves `/v1/exporter/decideds` via the v2 handler (adds `errors`).
+
+**Standard mode**
+- No duty tracer or trace persistence. Keeps legacy participants flow and response shape.
+- Background pruning based on `RetainSlots` (initial prune and then continuous, slot‑aligned).
+ - When the exporter is enabled, the node subscribes to all subnets unless you configure fixed subnets.
+
+### 2) Expose API ports
+
+```yaml
+# HTTP API (required for clients)
+SSVAPIPort: 16000
+
+# WebSocket API (optional; use a different port than SSVAPIPort)
+WebSocketAPIPort: 16001
+WithPing: true
+```
+
+> The OpenAPI spec is versioned under `docs/api/ssvnode.openapi.yaml|json`.
+
+### 3) Minimal working configs
+
+**Archive mode (recommended for full tracing)**
+
+```yaml
+global:
+  LogLevel: info
+
+db:
+  Path: ./data/db
+
+eth2:
+  BeaconNodeAddr: http://your-beacon:5052
+
+eth1:
+  ETH1Addr: ws://your-execution:8546/ws
+
+# p2p: optional fixed subnets. When the exporter is enabled (any mode),
+#      the node subscribes to all subnets to capture duties unless you set fixed subnets.
+
+exporter:
+  Enabled: true
+  Mode: archive
+
+SSVAPIPort: 16000
+WebSocketAPIPort: 16001
+WithPing: true
+```
+
+**Standard mode (v1‑compatible)**
+
+```yaml
+exporter:
+  Enabled: true
+  Mode: standard
+  RetainSlots: 50400
+
```

Comment:
why previously you included full config and here just relevant part? unify it to either. you can include the other part separately or just mention eth nodes have to be set like usual nodes.